OpenDreamShared/Compiler/Token.cs

```diff
@@ -29,6 +29,7 @@ public enum TokenType : byte {
         DM_Preproc_Pragma,
         DM_Preproc_Punctuator,
         DM_Preproc_Punctuator_Colon,
+        DM_Preproc_Punctuator_Semicolon,
```

Comment:
Keep this in alphabetical order, move this below `DM_Preproc_Punctuator_RightParenthesis`